your-firsts/src/create-fungible-token.ts

```diff
@@ -0,0 +1,17 @@
+import { web3 } from '@alephium/web3'
+import { testNodeWallet } from '@alephium/web3-test'
+import { ShinyToken } from '../artifacts/ts'
+
+export async function createFungibleToken(issueTokenTo?: string): Promise<string> {
+  web3.setCurrentNodeProvider('http://127.0.0.1:22973', undefined, fetch)
+  const signer = await testNodeWallet()
+  const issueTokenAmount = 10000n
+  // Deoloy `ShinyToken` contract and issue `10000` shiny tokens to `issueTokenTo` address.
+  const shinyToken = await ShinyToken.deploy(signer, {
```

Comment:
It might be confusing to educate new developers on deploying contracts using two different methods:
1. Raw approach: `ShinyToken.deploy(...)`
2. Using deploy scripts: `deployer.deployContract(...)`

But I don't have a better solution here.

Reply:
If we had to pick one maybe `deployer.deployContract` is better? Since it is probably what's gonna be used in production.

Reply:
But i tried to avoid deployment scripts in the example, since it is another concept. Same for the `AwesomeNFT` template id, it feels a lot easier that everything happens in one ts file.

Reply:
Let's keep it this way for now, I don't have any better ideas.